lib/testers/rule-tester.js

```diff
@@ -14,11 +14,12 @@ function expectEqual(a, b) {
   assert.deepStrictEqual(JSON.stringify(a, replacer, 2), JSON.stringify(b, replacer, 2));
 }
 
-
 function verify(ruleName, rule, testCases) {
-  const linterConfig = {
-    rules: { [ruleName]: 2 }
-  };
+  function getLinterConfig(config = {}) {
```

Comment:
Should we call this `createLinterConfig`? This is what it does, right?

Reply:
Alright.

Reply:
Done.